packages/react-components/src/components/resource-explorers/explorers/alarm-explorer/transformAlarmData.ts

```diff
@@ -0,0 +1,42 @@
+import { AlarmData, AlarmDataStatus } from '../../../../hooks/useAlarms';
+import { parseAlarmStateAssetProperty } from '../../../../hooks/useAlarms/transformers';
+import {
+  AlarmResource,
+  AlarmResourceWithLatestValue,
+} from '../../types/resources';
+
+export const transformAlarmData = ({
+  assetId,
+  assetModelId,
+  compositeModelId,
+  compositeModelName,
+  state,
+  status,
+}: AlarmData): {
+  resource: Partial<AlarmResourceWithLatestValue>;
+  status: AlarmDataStatus;
+} => {
+  const latestState = parseAlarmStateAssetProperty(state?.data?.at(-1));
+
+  return {
+    resource: {
```

Comment:
There should be a column with the name of the inputProperty if there is one

Reply:
Yes, I can add one! there should always be one actually.